.changeset/tough-ties-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cultureamp/rich-text-toolkit": patch
+---
+
+Update @kaizen dependencies
```

Comment:
Lol why didn't you list all the versions in this PR??

Reply:
This repo doesn't get my love